projects/gloo/cli/pkg/flagutils/install.go

```diff
@@ -11,3 +11,18 @@ func AddInstallFlags(set *pflag.FlagSet, install *options.Install) {
 	set.StringVarP(&install.HelmChartOverride, "file", "f", "", "Install Gloo from this Helm chart archive file rather than from a release")
 	set.StringVarP(&install.Namespace, "namespace", "n", defaults.GlooSystem, "namespace to install gloo into")
 }
+
+func AddKnativeInstallFlags(set *pflag.FlagSet, install *options.Knative) {
+	set.StringVarP(&install.InstallKnativeVersion, "install-knative-version", "v", "0.7.0",
+		"Version of Knative-Serving to install, when --install-knative is set to `true`")
```

Comment:
I think this version applies to all the components of knative right?
```suggestion
		"Version of Knative to install, when --install-knative is set to `true`")
```